sycamore/data/document.py

```diff
@@ -78,6 +78,14 @@ def embedding(self) -> Optional[list[float]]:
     def embedding(self, embedding: list[float]) -> None:
         self.data["embedding"] = embedding
 
+    @property
+    def simHashes(self) -> Optional[list[int]]:
```

Comment:
Do we want simHashes to be a top-level field in all Documents? I guess this is a bit like the question of what transforms to add to DocSet, but it's not immediately obvious to me why this should be top-level vs a property. If we do want to keep it at the top-level, do we want to call it something more generic (fingerprint, dedupe_val, ...) to allow for potentially other algorithms in the future, or do we think we would add another top-level field?

Reply:
I was thinking it's similar to embeddings, so followed that pattern.  I wanted to name it specifically so that there wouldn't be a name clash if we wanted to use another algorithm in the future.  The only major requirement is that they end up in OpenSearch in such a way that we can perform retrieval-time deduplication.